### ISSUE TITLE
Cut bug

### ISSUE THREAD

**Issue by Armaggik** [geoman-io/leaflet-geoman#630]
![Geoman](https://user-images.githubusercontent.com/68292888/87450814-35fdb680-c629-11ea-9cbc-42c1261b37a8.gif)

**Comment by Falke-Design** [comment]
@codeofsumit This is happening because of to much multi-polygon holes. When I flatten the multi-polygon to single layers it works

**Comment by codeofsumit** [comment]
I'm not aware of a limit in MultiPolygons. Is that a bug in Turf?

**Comment by Falke-Design** [comment]
Test what should be passed:

```
  it('test cut bug', ()=>{
    cy.toolbarButton('rectangle').click();
    cy.get(mapSelector)
      .click(191,216)
      .click(608,323);

    cy.toolbarButton('cut').click();
    cy.get(mapSelector)
      .click(226,389)
      .click(230,105)
      .click(270,396)
      .click(226,389);

    cy.toolbarButton('cut').click();
    cy.get(mapSelector)
      .click(293,356)
      .click(293,122)
      .click(340,367)
      .click(293,356);

    cy.toolbarButton('cut').click();
    cy.get(mapSelector)
      .click(364,345)
      .click(363,138)
      .click(414,368)
      .click(364,345);

    cy.toolbarButton('edit').click();
    cy.hasVertexMarkers(16);
  })
```

@codeofsumit yes it is because of Turf.js (https://github.com/Turfjs/turf/issues/1922)

**Comment by Falke-Design** [comment]
Fixed in Release:  `2.7.0`
